lib/chef/http/auth_credentials.rb

```diff
@@ -44,7 +45,7 @@ def signature_headers(request_params={})
         request_params                 = request_params.dup
         request_params[:timestamp]     = Time.now.utc.iso8601
         request_params[:user_id]       = client_name
-        request_params[:proto_version] = Chef::Config[:authentication_protocol_version]
+        request_params[:proto_version] = config[:authentication_protocol_version]
```

Comment:
I wonder if "latest" is still an allowed value? We should probably have this fallback to something like that to force upstream users to set an appropriate version for their apps (when using outside of Chef).